src/Components/ArticleCard.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { fetchData } from '../Utils/api';
+
+const ArticleCard =() => {
+    const { article_id } = useParams()
+    const [article, setArticle] = useState([])
+
+    useEffect(() => {
+        fetchData(`/articles/${article_id}`).then(({article})=>{
```

Comment:
Ideally we would separate out this endpoint into a utils.api and create a function that will call this end point for us.